chirp/drivers/kguv980p.py

```diff
@@ -1636,54 +1636,49 @@ def apply_psw_id(setting, obj):
         # Freq Limits settings
 
         # Convert Integer back to correct limit HEX value:
+        s = self._memobj
 
-        _temp = str(hex(self._memobj.bandlimits.limit_144M_ChA_rx_start))
-        _temp = int(_temp[2:])//10
+        _temp = int(s.bandlimits.limit_144M_ChA_rx_start) // 10
```

Comment:
So much better right?

Reply:
And I am learning about some more built in library functions.   I really like the bbcd data type.